
---
## Academic Qualifications
___

### PhD in Biology at the University of Sheffield (October 2015 - present)

"Towards understanding the mechanisms and forms of mutation in birds" 

### MBiolSci Biology at the University of Sheffield (September 2011 – June 2015) 

Awarded a 1st. 

### Maidstone Grammar School, Maidstone (September 2003 – June 2011)

A-Level Biology (A), Mathematics (A), and Physics (A), AS-Level Chemistry (B), and French (C), 11 GCSEs grades A*-B including English

___
## Research Experience 
___

### Postgraduate researcher (October 2015 - present)

* Worked with high-coverage whole genome resequencing data using a wide range of bioinformatics software including, GATK, SAMTools, PAML, MultiZ, BlastZ.
* Constructed custom bioinformatics pipelines
* Proficient in Python, R, Shell, commandline Linux and Git
* Written a number of custom Python modules: [anavar_utils](https://henryjuho.github.io/anavar_utils/), [WGAbed](https://henryjuho.github.io/WGAbed/), [python_qsub_wrapper](https://github.com/henryjuho/python_qsub_wrapper).
* Extensive experience working on the University of Sheffield's two high perfomance computing clusters 'ShARC' and 'iceberg'

### Masters research (September 2014 – June 2015)

Worked with demography inference software PSMC and MSMC to investigate the possibility of generating recombination maps
from a single genome with simulated data using ms, and compared the approach's application on the great tit reference genome
to the great tit linkage map.

### University of Sheffield Evolution and Behaviour Research Group (June – July 2013)

Awarded a Sheffield Undergraduate Research Experience studentship, supervised by Prof. 
Jon Slate. Investigated copy number variation (CNV) in wild great tit populations, 
using a custom 600k SNP chip. Work contributed to the ongoing annotation of the great 
tit genome project.

___
## Publications
___

**Barton, Henry J** & Zeng, Kai (2018b). The impact of natural selection on short insertion and deletion variation in the great tit genome. *Submitted*

**Barton, Henry J** & Zeng, Kai (2018a). New methods for inferring the distribution of fitness effects for INDELs and SNPs. *Molecular biology and evolution*, 35, 1536-1546.

Kim, J-M, Santure, AW, **Barton, Henry J**, Quinn, JL, Cole, Eleanor F, Great Tit HapMap Consortium, Visser, ME, Sheldon, BC, Groenen, MAM, van Oers, K & others (2018). A high-density SNP chip for genotyping great tit (Parus major) populations and its application to studying the genetic architecture of exploration behaviour. *Molecular ecology resources*, 9, 2987-3007.

Corcoran, Pádraic, Gossmann, Toni I, **Barton, Henry J**, Great Tit HapMap Consortium, Slate, Jon & Zeng, Kai (2017). Determinants of the efficacy of natural selection on coding and noncoding variability in two passerine species. *Genome Biology and Evolution*, 9, 2987-3007.

___
## Conference Presentations
___


* Evolution 2018 (August 2018), talk: ["Inferring the selective pressures acting on insertions and deletions in the great tit genome"](https://github.com/henryjuho/evolution2018_talk/blob/master/evolution2018_hbarton.pdf)

* Pop Group 51 (January 2018), talk: "The strength of GC biased gene conversion is conserved across two passerine species"

* Pop Group 50 (January 2017), talk: ["Insertions and deletions in the great tit genome"](https://github.com/henryjuho/PopGroup50_talk/blob/master/HBarton_popgroup50_pres.pdf) 

___
## Teaching experience
___

### Undergraduate parasitology practical (February 2018)

* Ran a first year undergraduate practical on investigating endoparsite abundance in sheep, including project design, lab work, statistical analysis in R and report writing.

### NERC Biomolecular Analysis Facility's Bioinformatics Course (January 2018)

* Taught a session on SNP calling for PhD students and postdoctoral researchers from external institutions.

### Departmental python course (May 2017) 

* Co-designed and taught a python course aimed at departmental researchers wanting to get started with python.

### Graduate teaching assistant (October 2015 - present)

* Over 240 hours demonstrating on undergraduate practicals over the course of the PhD

___
## References
___

Available on request